zppy/templates/ts.bash

```diff
@@ -109,7 +109,11 @@ cat input.txt | ncclimo \
 --dpf={{ dpf }} \
 --tpd={{ tpd }} \
 {%- endif %}
+{%- if input_files.split(".")[0] == 'cam' or input_files.split(".")[0] == 'eam' or input_files.split(".")[0] == 'elm' or input_files.split(".")[0] == 'clm2'  %}
+--prc_typ={{ input_files.split(".")[0][:3] }}
+{%- else %}
 --prc_typ={{ prc_typ }}
+{%- endif %}
```

Comment:
@BunnyVon @chengzhuzhang Does this still accomplish what #421 was trying to do? For reference, that change was:
```
--prc_typ={{ input_files.split(".")[0] }}
```
to
```
--prc_typ={{ input_files.split(".")[0][:3] }}
```

@golaz Does this still accomplish what #424 was trying to do? For reference, that change was:
```
{%- if input_files.split(".")[0] == 'cam' or input_files.split(".")[0] == 'eam' or input_files.split(".")[0] == 'elm' or input_files.split(".")[0] == 'clm2'  %}
--prc_typ={{ input_files.split(".")[0][:3] }}
{%- else %}
--prc_typ=sgs
{%- endif %}
```
to:
```
--prc_typ={{ prc_typ }}
```

Reply:
I think so. I was able to run the bash files by changing `--prc_typ=clm`

Reply:
@forsyth2 

I think for following the logic for https://github.com/E3SM-Project/zppy/pull/424, 
We can just modify the block here: https://github.com/E3SM-Project/zppy/blob/08838055a8ff26fc9404f87f7c73466ff04f8506/zppy/utils.py#L159-L161 to make sure the `prc_typ` for `clm2` should be `clm`.

Reply:
@chengzhuzhang I added a second commit with that change:
```
    elif tmp in ("clm2",):
        component = "lnd"
        prc_typ = "clm"
    elif tmp in ("elm",):
```
Is that correct? Or did we want it for `elm` too?